README.md

```diff
@@ -172,9 +170,21 @@ Application | GitHub Link | Store/Marketplace
 Visual Studio Code | [warpnet/vscode-salt-lint](https://github.com/warpnet/vscode-salt-lint) | [VisualStudio Marketplace](https://marketplace.visualstudio.com/items?itemName=warpnet.salt-lint)
 Sublime Text | [warpnet/SublimeLinter-salt-lint](https://github.com/warpnet/SublimeLinter-salt-lint) | [Package Control](https://packagecontrol.io/packages/SublimeLinter-contrib-salt-lint)
 
-
 Wish to create a `salt-lint` extension for your favourite editor? We're always looking for [contributions](CONTRIBUTING.md)!
 
+# Fix common issues
+
+`sed` might be one of the better tools to fix common issues.
+
+Fix spacing arround `{{ var_name }}`, eg. `{{env}}` --> `{{ env }}`:\
+`sed -i -E "s/\{\{\s?([^}]*[^} ])\s?\}\}/\{\{ \1 \}\}/g"  $(find salt -name '*.sls')`
+
+Make the `dir_mode`, `file_mode` and `mode` arguments in the desired syntax:\
+`sed -i -E "s/\b(dir_|file_|)mode: 0?([0-7]{3})/\1mode: '0\2'/"  $(find salt -name '*.sls')`
```

Comment:
Don't assume everyone's salt repo is called `salt`, instead assume they are inside their repository.

```suggestion
`sed -i -E "s/\b(dir_|file_|)mode: 0?([0-7]{3})/\1mode: '0\2'/"  $(find . -name '*.sls')`
```

Reply:
Fixed in #109